browser-test/src/support/ti_dashboard.ts

```diff
@@ -0,0 +1,54 @@
+import {Page} from 'playwright'
+
+export class TiDashboard {
+  public page!: Page
+
+  constructor(page: Page) {
+    this.page = page
+  }
+
+  async gotoTIDashboardPage(page: Page) {
+    await page.click('text="Trusted intermediary dashboard"')
+  }
+
+  async fillFormForNewClients(client: ClientInformation) {
```

Comment:
`fillForm` sounds like it only fills while your function also submits the form creating the client. So maybe rename it to `fillAndSubmitFormForNewClient` or `createClient`?